site/public/js/drag-and-drop.js

```diff
@@ -683,6 +684,21 @@ function handleBulk(gradeable_id, num_pages, use_qr_codes = false, qr_prefix = "
                 alert("ERROR! You may not use angle brackets in your filename: " + file_array[i][j].name);
                 return;
             }
+
+            total_size += file_array[i][j].size;
+
+            if (total_size >= max_file_size){
+                alert("ERROR! Uploaded file(s) exceed max file size.\n" + 
+                      "Please visit https://submitty.org/sysadmin/system_customization for configuration instructions.");
+                return;
+            }
+
+             if (total_size >= max_post_size){
+                alert("ERROR! Uploaded file(s) exceed max PHP POST size.\n" + 
+                      "Please visit https://submitty.org/sysadmin/system_customization for configuration instructions.");
+                return;
```

Comment:
Go through all of the error conditions in this loop and make sure that they have `$("#submit").prop("disabled", false);` before calling `return` in them so that the user is not stuck with a dead form.